bin/build-packages.js

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env node
+
+// find all the packages
+const glob = require('glob');
+const {execSync} = require('child_process');
+const path = require('path');
+
+glob( 'packages/*/package.json', ( err, matches ) => {
+	if ( err ) { console.error( err ); return; }
+
+	matches.forEach( match => {
+		const dir = path.dirname( match );
+		const inputDir = path.join( dir, 'src' );
+		const outputDir = path.join( dir, 'dist', 'esm' );
+		console.log( `Building ${ dir }...` );
+		execSync( `node ./node_modules/.bin/babel -d ${outputDir} ${inputDir}`, {
+			env: Object.assign( {}, process.env, { CALYPSO_CLIENT: 'true' } )
```

Comment:
This means transpile to ES5/ESM (modules), right? We should really rename that flag :slightly_frowning_face:

Reply:
yeah, it means a bunch of stuff right now.

Reply:
Maybe add an inline comment for the time being to explain it?